samples/template/README.rst

```diff
@@ -0,0 +1,305 @@
+         ``Sending`` pin is active when the packet is being transmitted.
+
+      * P1.4:
+         The ``Received`` pin is toggled ``on`` and immediately ``off`` when the packet has been received.
+
+   .. tab:: nRF5340
+
+      * P1.2:
+         ``DFU`` state.
+
+      * P1.3:
+         The ``Received`` pin is toggled ``on`` and immediately ``off`` when the packet has been received.
+
+      * P1.4:
+         ``Sending`` pin is active when the packet is being transmitted.
+
+      * P1.8:
+         ``ERROR`` has occured.
+
+.. _template_config:
+
+Configuration
+*************
+
+When running the sample, you can use different configurations files depending on the supported features.
+
+.. _template_build_type:
+
+Build types
+===========
+
+Configuration files are provided for different build types and are located in the application root directory.
+
+The :file:`prj.conf` file represents a ``debug`` build type.
+Other build types are covered by dedicated files with the respective build type added to the file name.
+For example, the ``release`` build type file is named :file:`prj_release.conf`.
+The same naming convention is followed if a board has other configuration files that are, for example, associated with the partition layout or child image configuration.
+
+Before you start testing the application, you can select one of the build types supported by the sample.
+Depending on the selected board, a sample supports the following build types:
+
+* ``debug`` -- Debug version of the application - can be used to enable additional features, such as logs or command-line shell, to verify the application behavior.
+* ``release`` -- Release version of the application - can be used to enable only the necessary application functionalities to optimize its performance.
+
+You can build the ``release`` firmware for ``build_target`` by running the following command in the project directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   $ west build -b *build_target* -- -DCONF_FILE=prj_release.conf
+
+For example:
+
+.. code-block:: console
+
+   $ west build -b nrf52840dk_nrf52840 -- -DCONF_FILE=prj_release.conf
+
+.. note::
+    Selecting a build type is optional.
+    However, if the build type is not selected, the ``debug`` build type is used by default.
+
+Command Line Interface
+======================
+
+The Sidewalk CLI can be set by the Kconfig
+
+CONFIG_SIDEWALK_CLI
+
+The SIDEWALK_CLI enables CLI compatible with sid_dut sample.
+
+To get list of supported CLI commands, you can use `sid help`.
+
+
+Power measurement test scenario
+===============================
+
+The application may perform the common Sidewalk application steps such us:
+
+#. Registration
+#. Time sync
+#. Send hello message to cloud
+#. Echo messages received from cloud
+
+This application is especially useful to test release configuration, and can be set by the Kconfig
+
+CONFIG_SIDEWALK_APP_ECHO
+
+You can specify the default transport protocol using Kconfig
```

Comment:
This akapit is somehow hidden.
In general I would suggest to create two section.
1. Application extensions
    1.1. Power app
    1.2. SensorDemo app
2. Configurations
     2.1. CLI
     2.2. Initial LINK_TYPE
     2.3. Minimal Sidewalk libraries
     2.4. DFU - if it will be selectable

Reply:
Please see updated version